docs/Cloud_integrations/Device_Integration_Center/integrations/influx.md

```diff
@@ -113,12 +113,12 @@ needs by editing the body template.
 <measurement>[,<tag_key>=<tag_value>] <field_key>=<field_value> [<timestamp>]
 ```
 
-!!! Note
+!!! Note "You should first define the body template"
 
-     When influx creates a measurement table, columns types are defined and all requests that don't match the type are 
+     When influx creates a measurement table, columns types are defined and all requests that don't match the type will be 
```

Comment:
I mean the previous version was OK. 0 conditional in such instances is preferable, as it shows what happens anytime the condition is met. It's like a fact or general rule, therefore I recommend using 0 conditional in similar sentences